managed/services/management/service.go

```diff
@@ -154,10 +154,10 @@ func (s *ServiceService) checkServiceType(service *models.Service, serviceType i
 
 func (s *ServiceService) validateRequest(request *managementpb.RemoveServiceRequest) error {
 	if request.ServiceName == "" && request.ServiceId == "" {
-		return status.Error(codes.InvalidArgument, "service_id or service_name expected")
+		return status.Error(codes.InvalidArgument, "We can't find service associated with the local node, service_id or service_name expected")
 	}
 	if request.ServiceName != "" && request.ServiceId != "" {
-		return status.Error(codes.InvalidArgument, "service_id or service_name expected; not both")
+		return status.Error(codes.InvalidArgument, "We can't find service associated with the local node, service_id or service_name expected; not both")
```

Comment:
these errors shouldn't be returned on the backend side, because it doesn't make sense for direct API requests. So it's better to wrap the error message returned by pmm-managed on the pmm-admin side.